nrf_wifi/fw_if/umac_if/src/default/fmac_api.c

```diff
@@ -413,6 +416,11 @@ enum nrf_wifi_status nrf_wifi_fmac_dev_init(struct nrf_wifi_fmac_dev_ctx *fmac_d
 			status = NRF_WIFI_STATUS_FAIL;
 			goto out;
 		}
+		nrf_wifi_init_ph_rf_params(&phy_rf_params, package_info);
```

Comment:
Shouldn't `package_info be read from somewhere before this ?

Reply:
As of now we have default support for QFN. Once we get the support for CSP package then we have to read the memory and update the package_info. So, currently no need to read it.